android/conductor/app/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.charlietuttle.conductor"
```

Comment:
Please change to org.androidtutorials.conductor. Also refactor all Java package names accordingly.